drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -189,3 +189,6 @@ validationSucceed=Validation Succeed
 
 backgroundTabTitle=Background
 backgroundErrorNotification=There are errors in Background data, please have a look
```

Comment:
Are you sure that we shell mention all of the limitation for expressions?
Bcs we are obviosly do not mention that if you will /0 the expression will fall, why should we mention about the return type? Just my feeling of the cheat sheet idea. I would like to only mention a key words.
**#** specifies expression you can use to invoke Java code.
**actualValue** specifies parameter for assertion in EXPECT.
But if you have other ideas...

If you really want to leave it here a shorten version of it as well:
Return type of the expression in GIVEN section should be the same as type of the column.
Return type of the expression in EXPECT section should be boolean assertion result.

Reply:
I can remove the additional information but I think it could be useful to have them here. I cannot user your suggested version because in EXPECT expression can return the same type (and in this case it will be verified as `==` to the actualValue) or a boolean so I mentioned both

Reply:
Ok, what about using /expression? Coz we are no need to put # before the expression for those column.